.ci/component_descriptor

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SOURCE_PATH="$(dirname $0)/.."
+COMMIT_SHA="$(git --git-dir ${SOURCE_PATH}/.git rev-parse HEAD)"
+
+if [[ ${RESOURCES_FILE_PATH} == "" ]]; then
+  RESOURCES_FILE_PATH="${SOURCE_PATH}/.landscaper/resources.yaml"
+fi
+
+printf "> Building Virtual Garden with version ${EFFECTIVE_VERSION} - ${COMMIT_SHA}\n"
+
+printf "> Create Virtual Garden ca archive\n"
```

Comment:
What is a "ca archive"?

Reply:
ca is a component archive (see https://gardener.github.io/component-spec/oci.html?highlight=archive). We change the text to `Create Virtual Garden Component Archive`.